operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2575,6 +2575,10 @@ compactor:
   env: []
   extraEnvFrom: []
 
+gossipRing:
```

Let's call it `gossip_ring` to be more consistent with how mimir own components are called

```suggestion
gossip_ring:
```